python/cugraph-pyg/cugraph_pyg/tests/conftest.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+
+import pytest
+
+from dask.distributed import Client
+from dask_cuda import LocalCUDACluster
+from dask_cuda.initialize import initialize
+
+from cugraph.dask.comms import comms as Comms
+from cugraph.dask.common.mg_utils import get_visible_devices
+
+
+# module-wide fixtures
+
+
+# Spoof the gpubenchmark fixture if it's not available so that asvdb and
+# rapids-pytest-benchmark do not need to be installed to run tests.
+if "gpubenchmark" not in globals():
+
+    def benchmark_func(func, *args, **kwargs):
+        return func(*args, **kwargs)
+
+    @pytest.fixture
+    def gpubenchmark():
+        return benchmark_func
+
+
+@pytest.fixture(scope="module")
+def dask_client():
+    dask_scheduler_file = os.environ.get("SCHEDULER_FILE")
+    cluster = None
+    client = None
+    tempdir_object = None
+
+    if dask_scheduler_file:
+        # Env var UCX_MAX_RNDV_RAILS=1 must be set too.
+        initialize(
```

Comment:
I know this was copied from existing fixtures elsewhere, but I think it might be better to just use all defaults for this call (cc @VibhuJawa). FYI [here's](https://github.com/rapidsai/cugraph/blob/c6028e1fb9c1dc07b5c1411ce5f6d7fd45ee7368/python/cugraph-service/server/cugraph_service_server/cugraph_handler.py#L395-L407) the latest configurations for multi-node and `LocalCUDACluster` we've been using to get the latest benchmark numbers (and we'll probably need to update the other fixtures as well).

Reply:
Yup, we should not enable anything here and stick to defaults.

Reply:
I fixed this in another upcoming PR but I can make the change here too.